packages/block-library/src/cover/test/edit.js

```diff
@@ -375,4 +378,76 @@ describe( 'Cover block', () => {
 			} );
 		} );
 	} );
+
+	describe( 'isDark settings', () => {
+		test( 'should set is-light class if light background set', async () => {
+			const { container } = await setup();
+			const colorPicker = screen.getByRole( 'button', {
+				name: 'Color: White',
+			} );
+			await userEvent.click( colorPicker );
+
+			const coverBlock = // eslint-disable-next-line testing-library/no-node-access
+				container.getElementsByClassName( 'wp-block-cover' );
+
+			expect( coverBlock[ 0 ] ).toHaveClass( `is-light` );
```

Comment:
Tiny code style nit: the `is-light` strings in this file use back-ticks instead of `'`. Since there's no string interpolation happening, should we switch to single quotes?

Reply:
thanks, fixed.